### PR TITLE
Magento_Multishipping: avoid using deprecated escape* methods from Ab…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/link.phtml [hunk 1]
```diff
@@ -5,4 +5,4 @@
  */
 
 ?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/link.phtml:10]: You missed defining phpdoc block for escaper here

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/overview.phtml [hunk 2]
```diff
@@ -4,7 +4,10 @@
  * See COPYING.txt for license details.
  */
 
-/** @var \Magento\Multishipping\Block\Checkout\Overview $block */
+/**
+ * @var \Magento\Multishipping\Block\Checkout\Overview $block
+ * @var \Magento\Framework\Escaper $escaper
+ */
 /** @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer */
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/overview.phtml]:
```suggestion
/**
 * @var \Magento\Multishipping\Block\Checkout\Overview $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
 */
```

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/overview/item.phtml [hunk 3]
```diff
@@ -12,43 +12,44 @@
  * Quote Item row html
  *
  * @see \Magento\Checkout\Block\Cart\Item\Renderer
+ * @var \Magento\Framework\Escaper $escaper
  */
 ?>
 <?php /** @var $block Magento\Checkout\Block\Cart\Item\Renderer */ ?>
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/overview/item.phtml]:
```suggestion
 * @var \Magento\Checkout\Block\Cart\Item\Renderer $block
 * @var \Magento\Framework\Escaper $escaper
 */
?>
```

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/shipping.phtml [hunk 4]
```diff
@@ -12,18 +12,19 @@
  * Multishipping checkout shipping template
  *
  * @var $block \Magento\Multishipping\Block\Checkout\Shipping
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/shipping.phtml]:
```suggestion
 * @var \Magento\Multishipping\Block\Checkout\Shipping $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/state.phtml [hunk 5]
```diff
@@ -12,12 +12,13 @@
  * Mustishipping state
  *
  * @see \Magento\Multishipping\Block\Checkout\State
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/state.phtml]:
```suggestion
 * @var \Magento\Multishipping\Block\Checkout\State $block
```

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml [hunk 6]
```diff
@@ -12,89 +12,90 @@
  * Ship to multiple address template
  *
  * @var $block \Magento\Multishipping\Block\Checkout\Addresses
+ * @var \Magento\Framework\Escaper $escaper
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml]:
```suggestion
 * @var \Magento\Multishipping\Block\Checkout\Addresses $block
 * @var \Magento\Framework\Escaper $escaper
```

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/billing.phtml [hunk 7]
```diff
@@ -8,13 +8,14 @@
  * Multishipping checkout billing information
  *
  * @var $block \Magento\Multishipping\Block\Checkout\Billing
+ * @var \Magento\Framework\Escaper $escaper
  * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
```
Comment (ihor-sviziev) [inline app/code/Magento/Multishipping/view/frontend/templates/checkout/billing.phtml]:
```suggestion
 * @var \Magento\Multishipping\Block\Checkout\Billing $block
 * @var \Magento\Framework\Escaper $escaper
 * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
```

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/overview.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/overview.phtml]: fixed

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/overview/item.phtml (hunk 3 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/overview/item.phtml]: fixed

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/state.phtml (hunk 5 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/state.phtml]: fixed

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml (hunk 6 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/addresses.phtml]: fixed

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/billing.phtml (hunk 7 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/billing.phtml]: fixed

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/shipping.phtml (hunk 4 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/shipping.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Database Compare, Functional Tests B2B, Functional Tests EE, Static Tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Please resolve conflict

**Comment by sergeynezbritskiy** [comment]
> Please resolve conflict

Hi @ihor-sviziev, fixed

File: app/code/Magento/Multishipping/view/frontend/templates/checkout/link.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/Multishipping/view/frontend/templates/checkout/link.phtml:10]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Database Compare, Functional Tests B2B

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes from this PR.

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-8905](https://jira.corp.magento.com/browse/ENGCOM-8905) has been created to process this Pull Request

**Pull request closed without merging**